### PR TITLE
Check user type and add open Azure ticket example

### PR DESCRIPTION
**Pull request by fisher046**
1. Check if the user type is Member.
2. Add an example for opening Azure ticket.

### CODE REVIEW

File: azure/azure-preflight.sh [hunk 1]
```diff
@@ -264,6 +264,18 @@ function validate_role_assignment() {
 
 validate_role_assignment "$account"
 
+#### Azure User Type Checking
+function validate_user_type() {
+  user_type=$(az ad signed-in-user show -o json | jq .userType | tr -d '"')
```
Comment (Simonzhaohui) [inline azure/azure-preflight.sh:269]:
```
$ az ad signed-in-user show --query userType -o tsv
Member
```

File: azure/azure-preflight.sh [hunk 2]
```diff
@@ -264,6 +264,18 @@ function validate_role_assignment() {
...
+  if [ "$user_type" != "Member" ]; then
+    msg="Current user is a $user_type user, not Member user"
+    store_suggestion "$msg"
+    suggest "$msg" alert
```
Comment (xugy99) [inline azure/azure-preflight.sh:273]: we need to remove this line temporarily as it breaks the colorful output on some platform.

File: azure/azure-preflight.sh (hunk 1 above)
Comment (fisher046) [reply azure/azure-preflight.sh]: Changed.

File: azure/azure-preflight.sh (hunk 2 above)
Comment (fisher046) [reply azure/azure-preflight.sh]: Changed.

**Review by xugy99 — APPROVED** [review]

**Pull request merged**